packages/frontend/components/primitives/category-badge/component.tsx

```diff
@@ -0,0 +1,43 @@
+﻿import { IconColor, IconName } from '@enums';
+import { Container, IconButton } from '@primitives';
+import { Splide, SplideSlide } from '@splidejs/react-splide';
+import * as styles from './styles';
+
+interface CategoryBadgesProps {
+  badges: {
+    name: string;
+    onClick?: () => void;
+  }[];
+}
+
+export const CategoryBadges = ({ badges }: CategoryBadgesProps) => (
+  <Container>
+    <Splide
+      options={{
+        autoWidth: true,
+        pagination: false,
+        arrows: false,
+        wheel: true,
+        releaseWheel: true,
+        mediaQuery: 'max',
+        breakpoints: {},
+      }}
+    >
+      {badges.map((badge) => (
+        <SplideSlide key={badge.name}>
+          <div css={styles.badge} key={badge.name}>
+            {badge.name}
+            <IconButton
+              icon={IconName.XMARK}
+              size="xs"
+              onClick={badge.onClick}
+              color={IconColor.GRAY}
+              ariaLabel={badge.name + '_xmark'}
```

Comment:
It describes what this button does for screen reader. Use localisation here.

Reply:
like: t('remove', { param  }). 

in localisation: remove filter {{param}}

Reply:
done